proxy.config.json

```diff
@@ -5,5 +5,12 @@
     "pathRewrite": {
       "^/api": ""
     }
+  },
+  "/notes": {
```

Comment:
tbh I think we should get rid of this file, as the main idea is to not to use the proxy at all in prod and as such it would be good to work without it locally.